docs/DEVELOPMENT.md

```diff
@@ -178,6 +178,26 @@ struct User {
 
 then simply run `cargo test`. The generated bindings are placed in `src-tauri/bindings/`. Make sure to add and commit them as well!
 
+### Internationalization
+
+For FlightCore to be used by the largest number, its interface is translated in several languages; users can choose used language through FlightCore settings.
+
+Localization files are located in `src-vue/src/i18n/lang`.
```

Comment:
Would also be nice to know how to add an additional language. Is it just adding a file with the right name, e.g. `de.ts` for German or is there more to it? ^^